python/cuml/experimental/fil/fil.pyx

```diff
@@ -1228,3 +1231,54 @@ class ForestInference(UniversalBase, CMajorInputTagMixin):
         return self.forest.predict(
             X, predict_type="per_tree", preds=preds, chunk_size=chunk_size
         )
+
+    @nvtx.annotate(
+        message='ForestInference.predict_per_tree',
+        domain='cuml_python'
+    )
+    def predict_leaf(
```

Comment:
Doesn't sklearn call this `apply`? If so, it's probably a good idea to remain consistent with sklearn.

Reply:
Personally, I feel that `apply()` is not very clear name

Reply:
Renamed the function to `apply()`